projects/plugins/jetpack/modules/sso.php

```diff
@@ -115,6 +118,32 @@ public function handle_invitation_results() {
 		}
 	}
 
+	/**
+	 * Renders errors/success messages for invite revocations.
+	 */
+	public function handle_revoke_invite_results() {
+		$valid_nonce = isset( $_GET['_wpnonce'] ) ? wp_verify_nonce( $_GET['_wpnonce'], 'jetpack-sso-revoke-user-invite' ) : false; // phpcs:ignore WordPress.Security.ValidatedSanitizedInput -- WP core doesn't pre-sanitize nonces either.
+
+		if ( ! isset( $_GET['jetpack-sso-revoke-user-invite'] ) || ! $valid_nonce ) {
+			return;
+		}
+		if ( $_GET['jetpack-sso-revoke-user-invite'] === 'success' ) {
+			return wp_admin_notice( __( 'User invite revoked successfully.', 'jetpack' ), array( 'type' => 'success' ) );
+		}
+		if ( $_GET['jetpack-sso-revoke-user-invite'] === 'failed' && isset( $_GET['jetpack-sso-revoke-user-invite-error'] ) ) {
+			switch ( $_GET['jetpack-sso-revoke-user-invite-error'] ) {
+				case 'invalid-user':
+					return wp_admin_notice( __( 'Tried to revoke an invite for a user that doesn&#8217;t exist.', 'jetpack' ), array( 'type' => 'error' ) );
+				case 'invalid-email':
+					return wp_admin_notice( __( 'Tried to revoke an invite for a user that doesn&#8217;t have an email address.', 'jetpack' ), array( 'type' => 'error' ) );
+				case 'invalid-invite-revocation':
+					return wp_admin_notice( __( 'Tried to revoke an invite that doesn&#8217;t exist.', 'jetpack' ), array( 'type' => 'error' ) );
+				case 'invalid-permissions':
+					return wp_admin_notice( __( 'You don&#8217;t have permission to revoke invites.', 'jetpack' ), array( 'type' => 'error' ) );
```

Comment:
We probably only need to add these two in the other handler that I have added. Then we can delete this function.